client/utils/session.js

```diff
@@ -1,4 +1,11 @@
 const handleSession = (orgSlug, token, cookies) => {
+  /*
+  This function is created while implementation of remember me checkbox
```

Comment:
`created while implementation` looks incorrect, what did you want to express here?